test/functional/test_runner.py

```diff
@@ -364,9 +364,10 @@ def main():
 def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=False, args=None, combined_logs_len=0, failfast=False, runs_ci, use_term_control):
     args = args or []
 
-    # Warn if bitcoind is already running (unix only)
+    # Warn if bitcoind is already running
+    # On macOS pidof (installed via brew) returns b''
```

Comment:
Is this a change that happened due to pidof change or due to a python change?

What does `pidof bitoind; echo $?` print?

Reply:
> Is this a change that happened due to pidof change or due to a python change?

The `pidof` source used by brew hasn't changed since 2005, so I'd guess a Python change if anything.

> What does pidof bitoind; echo $? print?

```bash
pidof bitcoind; echo $?
65657 
0
```

Reply:
I mean for the case where `bitcoind` is not running. :)

Reply:
For reference I get:

```
$ pidof bitoind; echo $?
1

Reply:
I get:
```bash
$ pidof bitcoind; echo $?
0
```

Reply:
Ok, so the issue is with the macos pidof